<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>12</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>34452</width>
      <height>32436</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFindback">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>4536765</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>找回密码</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSignup">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>14</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16666</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>234626</width>
      <height>23435</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLogin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>2435467</width>
      <height>765423</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
     </font>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSettings">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>3456784</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>网络设置</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../ILLW_SOURCE/head.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Settings" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditIP">
      <property name="maximumSize">
       <size>
        <width>34452</width>
        <height>32436</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPort">
      <property name="maximumSize">
       <size>
        <width>234626</width>
        <height>23435</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>8888</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="buttonSave">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2435467</width>
        <height>765423</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>保存设置</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>buttonSave</zorder>
   <zorder>lineEditPort</zorder>
   <zorder>lineEditIP</zorder>
   <zorder>widget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
